packages/markdown/processor/parse/flavored/callout.js

```diff
@@ -12,15 +12,19 @@ function tokenizer(eat, value) {
   title = title.trim();
 
   const style = {
-    '📘': 'info',
-    '⚠️': 'warn',
-    '🚧': 'warn',
-    '👍': 'okay',
-    '✅': 'okay',
-    '❗️': 'error',
-    '🛑': 'error',
-    ℹ️: 'info',
-    '⚠': 'warn',
+    '\uD83D\uDCD8': 'info',
+    '\u26A0\uFE0F': 'warn',
+    '\uD83D\uDEA7': 'warn',
+    '\uD83D\uDC4D': 'okay',
+    '\u2705': 'okay',
+    '\u2757': 'error',
+    '\u2757\uFE0F': 'error',
+    '\uD83D\uDED1': 'error',
+    '\u2049\uFE0F': 'error',
+    '\u203C\uFE0F': 'error',
+    // eslint-disable-next-line prettier/prettier
```

Comment:
What's this disabling?

Reply:
ohhh shit, so I meant to leave a comment asking you this exact thing.

Reply:
so whenever I run `npm test -- -u` on the repo, that line auto-prettifies itself _back_ to the ℹ️ emoji! I'm not quite sure what prettier rule it is that's catching this; maybe it's just some funky VSC setting on my machine? But my fear is that even with this disable comment, anyone who updates the snapshots for the repo will break the next release build in IE again without really noticing!

Reply:
Huh, yeah. Weird.

![Screen Shot 2020-05-07 at 9 03 05 AM](https://user-images.githubusercontent.com/33762/81317354-97117880-9041-11ea-9326-0e68f9b95c34.png)

Reply:
Leaving the Prettier exclusion in should be fine for future builds, but yeah that is concerning because:

```
> '\u2139\uFE0F' === 'ℹ️'
true
```